Formula/detekt.rb

```diff
@@ -18,7 +18,8 @@ class Detekt < Formula
 
   def install
     libexec.install "detekt-cli-#{version}-all.jar"
-    bin.write_jar_script libexec/"detekt-cli-#{version}-all.jar", "detekt", java_version: "17"
+    # remove after https://github.com/detekt/detekt/issues/5576
```

Comment:
Actually this would be better
```suggestion
    # remove `--add-opens` after https://github.com/detekt/detekt/issues/5576
```